README.md

```diff
@@ -233,16 +233,22 @@ git subtree add --squash --prefix=vendor/github.com/orgname/xyzzy/ vendor-xyzzy
 The [`scripts/presubmit.sh`](scripts/presubmit.sh) script runs various tools
 and tests over the codebase.
 
+#### Install [golangci-lint](https://github.com/golangci/golangci-lint#local-installation).
 ```bash
-# Install gometalinter and all linters
-go get -u github.com/alecthomas/gometalinter
-gometalinter --install
+go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+cd $(go env GOPATH)/src/github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
(I thought I made this comment on a previous iteration but GitHub ate it)

Maybe just `$GOPATH` rather than `${go env GOPATH)` as it's shorter and more consistent?